dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops

```diff
@@ -217,8 +217,15 @@ CWWKD1018.confl.special.param.useraction=Update the repository to not use \
  both of the Jakarta Data special parameter types in the same method signature.
 
 CWWKD1019.mixed.positional.named=CWWKD1019E: The {0} method of the {1} repository \
- interface cannot have a query with a mixture of positional parameters and named \
- parameters.
+ interface defines a mixture of {2} positional parameters and \
+ {3} named parameters ({4}) for the Query annotation value: {5}. A query with \
+ positional parameters must reference each parameter by its ordinal value, \
+ starting with ?1, and the method parameters must not have the Param annotation. \
+ A query with named parameters must reference each parameter by its name, \
+ such as {6}, and the method parameters must have the Param annotation, \
+ such as {7}. Alternatively, if you enable the -parameters compile option, \
+ the method parameters can indicate the named parameter by having the same name \
+ as the named parameter. For example, {8}.
```

Comment:
The {0} method in the {1} repository interface cannot have a query that mixes positional and named parameters. The interface defines a mix of {2} positional parameters and {3} named parameters {4} for the Query annotation value: {5}. A query with positional parameters must reference each parameter by its ordinal value, starting with ?1, and the method parameters must not have the Param annotation. In a query with named parameters, each parameter must be referenced by its name, such as {6}, and the method parameters must have the Param annotation, such as {7}. Alternatively, enabling the -parameters compile option allows method parameters to indicate the named parameter by matching its name, for example, {8}.

Reply:
updating as suggested, except for the change from of to in, which would be inconsistent with all of the other use in this file, as described in the other review comment